docs/guide/ecosystem/documentation.md

```diff
@@ -203,25 +200,48 @@ props: {
 #### Cons
 
 - Static code samples
+- Unsupported features of VueJs
```

Comment:
Maybe list which ones or rephrase like 
`Not fully compliant with all Vue.js specs` ?

Reply:
Indeed, I added some examples